pinot-core/src/main/java/org/apache/pinot/core/data/table/Record.java

```diff
@@ -22,33 +22,24 @@
 
 
 /**
- * Defines a single record in Pinot comprising of keys and values
+ * Defines a single record in Pinot
  */
 public class Record {
-  private Key _key;
-  private Object[] _values;
+  private Object[] _columns;
```

Comment:
Recommend renaming this to `_values`, `_columns` sounds like column names

Reply:
done